app/main/views/suppliers.py

```diff
@@ -165,75 +165,65 @@ def edit_registered_address():
 
 @main.route('/edit', methods=['GET'])
 @login_required
-def edit_supplier_redirect():
+def edit_what_buyers_will_see_redirect():
     # redirect old route for this view
-    return redirect(url_for('.edit_supplier'), 302)
+    return redirect(url_for('.edit_what_buyers_will_see'), 302)
 
 
-@main.route('/details/edit', methods=['GET'])
+@main.route('/what-buyers-will-see/edit', methods=['GET', 'POST'])
```

Comment:
Do we need to change this URL? Will it upset Ash with regards to analytics?

Reply:
Good point with analytics, I will ask Ash. Do you think we should leave it as was?

Reply:
Ash says it is ok as long as we give him the new url. @TheDoubleK is it ok to change this route? If we do, do we need to do anything else?

Reply:
I think it should be fine... the functional tests just click through the journey I think, without explicitly checking the URLs.